### PR TITLE
CNV-19923: remove the module about configuring file system overhead

### LINKED ISSUE #40119: [enterprise-4.9] Issue in file virt/virtual_machines/virtual_disks/virt-reserving-pvc-space-fs-overhead.adoc
### Which section(s) is the issue in?
Overriding the default file system overhead value

### What needs fixing?
It is not possible to override the default file system overhead value since OpenShift Virtualization (CNV) v4.8
So the doc should be changed for 4.8, 4.9, and upcoming 4.10 and 4.11

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.9, 4.10

Issue: [CNV-19923](https://issues.redhat.com//browse/CNV-19923)

Link to docs preview: http://file.rdu.redhat.com/sjess/CNV19923-1/virt/about-virt.html 

QE review: Kevin Goldblatt
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/virt-2-6-release-notes.adoc [hunk 1]
```diff
@@ -66,7 +66,7 @@ The SVVP Certification applies to:
 [id="virt-2-6-storage-new"]
 === Storage
 
-* When you add a virtual machine disk to a persistent volume claim (PVC) that uses the `Filesystem` volume mode, the Containerized Data Importer (CDI) now reserves 5.5% of the PVC space for file system overhead. If the default value is not ideal for your use case, you can xref:../virt/virtual_machines/virtual_disks/virt-reserving-pvc-space-fs-overhead.adoc#virt-reserving-pvc-space-fs-overhead[change the percentage] that CDI reserves for this purpose.
+* When you add a virtual machine disk to a persistent volume claim (PVC) that uses the `Filesystem` volume mode, the Containerized Data Importer (CDI) now reserves 5.5% of the PVC space for file system overhead.
```
Comment (maya-r) [inline virt/virt-2-6-release-notes.adoc:69]: We probably shouldn't edit 2.6 release notes

Comment (sabrinajess) [reply virt/virt-2-6-release-notes.adoc:69]: Eek that was unintentional. Thanks for the catch @maya-r

**Comment by maya-r** [comment]
lgtm

**Review by kgoldbla — APPROVED** [review]
lgtm

**Review by aireilly — COMMENTED** [review]
LGTM - However, I don't see a preview for this change, can you post the preview link.

**Comment by sabrinajess** [comment]
@aireilly Preview added (no direct link since PR is for removing content).

**Comment by sabrinajess** [comment]
@aireilly Is this OK to move to merge review?

**Comment by aireilly** [comment]
Sorry! Yep go for it.

**Pull request merged**